docs/getting-started-with-sylius/first-product.rst

```diff
@@ -0,0 +1,41 @@
+First product
+=============
+
+We move to one of the most important sections of the Admin panel - products management. As you can see, the **Catalog**
+section in the menu is quite extended, but we will, for now, focus on product creation only.
+
+You can pick a *simple* or *configurable* product type before its creation. Making long story short -
+*simple* is a product that has just one version,
+*configurable* is a product where the customer gets to choose some options of it (like size or colour).
+Check out the :doc:`Products </book/products/products>` chapter in **The Book** for more information.
+
+.. image:: /_images/getting-started-with-sylius/product-types.png
+    :scale: 55%
+    :align: center
+
+|
+
+During the product creation, you can decide about many of its traits. How much does it cost? Is it a physical product that requires
+shipping? Should it be tracked within the inventory system? On which channel will it be available? Should it have some taxes added during
+the checkout or not? Take a while to explore these options later, right now let's fill only the most critical data:
+
+.. image:: /_images/getting-started-with-sylius/product-creation.png
+
+Summary
+-------
+
+Great, the first stage is done! The whole checkout process is described in :doc:`this part of the documentation </book/orders/checkout>`,
+it's quite natural and logical, so it should not be a problem to understand it.
```

Comment:
`so it should not be a problem to understand it.` I would be afraid of using this phrase, as if someone will not understand it, they may feel dumb
```suggestion

```